src/app/[lng]/(main)/layout.tsx

```diff
@@ -8,10 +13,29 @@ export default async function MainLayout({
   children: React.ReactNode;
   params: { lng: string };
 }) {
+  const partnerThemes = await getPartnerThemes();
+  const cookies1 = cookies();
+
+  const defaultTheme = 'op';
+
   return (
-    <>
-      {children}
-      <FeatureCards />
-    </>
+    <ThemeProvider
+      themes={[
+        'dark',
+        'light',
+        ...partnerThemes.data.map((d) => d.attributes.uid),
```

Comment:
what is that array doing?

Reply:
taking the uid of all the dynamic themes and adding it to the themes array, to make them available across the app